src/client/sandbox/shadow-ui.js

```diff
@@ -188,12 +182,11 @@ export default class ShadowUI extends SandboxBase {
             });
 
             var restoreStyle = e => {
-                if (!this.getUIStylesheet()) {
-                    var headElemenet = e.document.head;
+                if (!this.isUIStylesheetsExists()) {
```

Comment:
`isUIStylesheetExists` I think, we can avoid using plural here I suppose. \cc @VasilyStrelyaev